azure-quantum/azure/quantum/storage.py

```diff
@@ -53,14 +54,7 @@ def get_container_uri(connection_string: str, container_name: str) -> str:
     return uri
 
 
-def upload_blob(
-    container: ContainerClient,
-    blob_name: str,
-    content_type:str,
-    content_encoding:str,
-    data: Any,
-    return_sas_token: bool=True
-) -> str:
+def upload_blob(container: ContainerClient, blob_name: str, content_type:str, content_encoding:str, data: Any, returnSasToken: bool=True) -> str:
```

Comment:
Why use camel case for `returnSasToken`? [PEP8 ](https://pep8.org/)recommends  using snake case.

Reply:
Form of habit from writing Typescript :)